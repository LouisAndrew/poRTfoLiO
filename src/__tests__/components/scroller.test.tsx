
import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

// import { render, cleanup } from '@testing-library/react'
import { cleanup } from '@testing-library/react'
import '@testing-library/jest-dom'

import Scroller  from 'components/scroller'

describe('Scroller', () => {
    const Element = <Scroller />

    afterEach(cleanup)

    it('renders without crashing', () => {
		const div = document.createElement('div')
		ReactDOM.render(Element, div)
	})
	
	/* it('renders correctly', () => {
		const { getByTestId } = render()
	}) */

	it('matches snapshot', () => {
		const run = false
	    
        if (run) {
	        const tree = renderer.create(Element).toJSON()
	        expect(tree).toMatchSnapshot()
	    }
	})
})
