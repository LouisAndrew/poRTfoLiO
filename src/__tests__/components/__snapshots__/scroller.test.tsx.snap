// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroller matches snapshot 1`] = `
<a
  href="/#projects"
>
  <div
    className="scroller css-1sfhpft"
    role="complementary"
  >
    <h4
      className="css-vurnku"
    >
      SCROLL DOWN TO SEE MY PROJECTS
    </h4>
    <div
      className="scroller__box css-1o8ebip"
      role="button"
    >
      <div
        className="scroller__thumb css-16w70n9"
        data-testid="scroller-thumb"
      />
    </div>
  </div>
</a>
`;
